build/build.xml

```diff
@@ -20,7 +20,7 @@
 	<property name="zap.download.url" value="https://github.com/zaproxy/zap-extensions/releases/download/2.7" />
 
 	<path id="build.classpath">
-		   <fileset dir="${build.lib.dir}" includes="*.jar"/>
+		<fileset dir="${build.lib.dir}" includes="*.jar"/>
```

Comment:
It would be nice if all these white space changes could be reverted so that the actual relevant change(s) stands out.
https://github.com/zaproxy/zaproxy/wiki/DevGuidelines#formatting